package.json

```diff
@@ -22,9 +22,9 @@
     "storybook:build": "storybook build"
   },
   "dependencies": {
-    "@mantine/core": "^7.13.4",
-    "@mantine/dates": "^7.13.4",
-    "@mantine/hooks": "^7.13.4",
+    "@mantine/core": "7.13.4",
+    "@mantine/dates": "7.13.4",
+    "@mantine/hooks": "7.13.4",
```

Comment:
locking mantine dependencies because github workflow complains when they get out of sync